DEPLOYMENT.md

```diff
@@ -0,0 +1,80 @@
+# Airtable Gateway deployment guide
+
+
+We at MANYC are presently only supporting [paid Heroku dynos](https://www.heroku.com/pricing).
+This project is an 'active' server that actively polls your airtable every 10 seconds. Unfortunately  the free Heroku tier won't let you poll for any period of time before the free Heroku dyno goes to [sleep](https://devcenter.heroku.com/articles/free-dyno-hours#dyno-sleeping)
+
+We hope in the future to have FULL deployment instructions on alternative platforms.
+
+Should you wish to **_test_** Airtable Gateway in a non-production scenario, follow the [Deploy to Heroku](#deploy-to-heroku) steps skipping the final steps (10-13) then follow the [additional steps](#additional-directions-for-a-non-production-free-dyno-to-test-with) after the main deployment instructions.
+
+## Deploy to Heroku
+
+Follow the steps below, but here is a video, with covering the first 9 steps if you get stuck: [How to deploy Airtable Gateway to Heroku](https://youtu.be/LHQ_xRf9Uc8). After you watch, resume with step 10.
+
+1. Contact MANYC to obtain three pieces of information:
+   1. MANYC New Request URL
+   2. MANYC Update Request URL
+   3. MANY Delete Request URL
+2. Add to your list of statuses a status 'Recalled' this will be used when MANYC triggers a cancel of a request.
+3. Click this button, and continue reading.
+   [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
+4. Give your app a unique Heroku name, the name itself doesn't have an effect.
+5. Choose the 'App Owner'
+6. For region, 'United States' is sufficient.
+7. Next up we have a quantity of 'Config Vars' to fill in. These are in alphabetical order. I will briefly describe them by their prefixes.
+   1. AIRTABLE_
```

Comment:
Just a typo having this underscores after the variable?

Reply:
No, intentional